rskj-core/src/main/java/co/rsk/peg/FederationProvider.java

```diff
@@ -28,9 +31,14 @@
  */
 public interface FederationProvider {
     // The currently "active" federation
-    Federation getActiveFederation();
+    Optional<Federation> getActiveFederation();
+    // The currently "active" federation's address
+    Optional<Address> getActiveFederationAddress();
```

Comment:
Is there any case when the active federation would not exist?

Reply:
Yes, updated.